### ISSUE TITLE
Markdown 3.0 installation causes startup exception

### ISSUE THREAD

**Issue by michaelsteigman** [pinax/pinax-blog#117]
Markdown 3.0 was release a few days ago. Due to the `>=` requirement for Markdown, it is installed now by default.  `ImagePattern`, which pinax-blog imports, has been removed. The result, for sites using `pinax-blog` deploying a new instance of their application is the following fatal exception during startup:

```
File "/opt/app-root/lib/python3.6/site-packages/pinax/blog/urls.py", line 4, in <module>
 from .views import (
File "/opt/app-root/lib/python3.6/site-packages/pinax/blog/views.py", line 25, in <module>
from .parsers.markdown_parser import parse
File "/opt/app-root/lib/python3.6/site-packages/pinax/blog/parsers/markdown_parser.py", line 2, in <module>
 from markdown.inlinepatterns import IMAGE_LINK_RE, ImagePattern
 ImportError: cannot import name 'ImagePattern'
```

**Comment by amhazeem** [comment]
This can be fixed by uninstalling markdown 3.0 

`pip uninstall markdown`

Then install an earlier version

`pip install Markdown==2.6.11`

This worked for me

**Comment by doktormerlin** [comment]
I created pull request https://github.com/pinax/pinax-blog/pull/123 for this

For future people having the same problem:

Instead of uninstalling Markdown first, you can just use `pip install -U markdown==2.6.11` for a step less

**Comment by KatherineMichel** [comment]
Closed by #123
